mitosheet/mitosheet/tests/test_mito_config.py

```diff
@@ -14,48 +14,88 @@ def delete_env_var_if_exists(env_var: str) -> None:
     if os.environ.get(env_var) is not None:
         del os.environ[env_var]
 
+def delete_all_mito_config_environment_variables() -> None:
+    """
+    Deletes all of the MITO CONFIG environment variables that are used in the MEC
+    """
+    delete_env_var_if_exists(MITO_CONFIG_KEY_VERSION)
+    delete_env_var_if_exists(MITO_CONFIG_KEY_SUPPORT_EMAIL)
+    delete_env_var_if_exists(MITO_CONFIG_KEY_CODE_SNIPPETS_SUPPORT_EMAIL)
+    delete_env_var_if_exists(MITO_CONFIG_KEY_CODE_SNIPPETS_URL)
+    delete_env_var_if_exists(MITO_CONFIG_KEY_CODE_SNIPPETS_VERSION)
```

Comment:
Let's write a loop over the list in the config class, so we can stop updating this